modules/app/components/InternalLink.tsx

```diff
@@ -33,7 +33,8 @@ export const InternalLink = ({
     scroll={scroll}
     title={title}
     sx={{ textDecoration: 'none', color: 'inherit', ...styles }}
+    passHref
   >
-    {children}
+    <a sx={{ textDecoration: 'none', color: 'inherit', ...styles }}>{children}</a>
```

Comment:
this sx property on a "non theme-ui" element causes the linter to fail, i tried solving it before but the only way i saw was either just using theme-ui elements or adding a tslint ignore on top of the file